spec-0012/index.md

```diff
@@ -4,122 +4,180 @@ number: 12
+   over `(u**v) + (y**z)`, and prefer `x + y + z` over `(x + y) + z`.
+1. Always use the `**` operator and unary `+`, `-`, and `~` operators *without*
+   surrounding whitespace. For example, prefer `-x**4` over `- (x ** 4)`. 
+2. Always surround non-PEMDAS operators with whitespace, and always make the priority of
+   non-PEMDAS operators explicit, even if the parentheses are consistent with the
+   implicit order of operations. For example, prefer `(x == y) or (w == t)` over
+   `x==y or w==t`.[^1]
+3. Always surround AS operators with whitespace.
+4. Typically surround MD operators with whitespace, except when there are lower-priority 
+   operators (namely AS) within the same compound expression or the division operation
+   would be written mathematically as a fraction with a horizontal bar.
+   For example, prefer `z = -x * y**t` over `z = -x*y**t`, but
+   prefer `z = w + x*y**t` over `z = w + x * y**t` due to the presence of the
+   lower-priority addition operator. Also, prefer `z = t/v * x/y` over
+   `z = t / v * x / y` if this would  be written mathematically as the product of
+   two fractions.
+5. According to the other rules, only `**`, `*`, `/`, and the unary `+`, `-`, and `~`
+   operators can appear in implicit subexpressions without spaces. In such expressions,
+   - Include unary operators at the left, and use at most one within an implicit
+     subexpression without spaces. 
+   - Include `**` operators at the right, and use at most one within an implicit
+     subexpression without spaces.
+  
+   To achieve these goals, simplification or the addition of parentheses may be required.
+   For example:
+   - The expressions `--x` and `-~x` would be implicit subexpressions without spaces
+     containing more than one unary operator. The former can be simplified to `+x` or
+     simply `x`, and the latter requires explicit parentheses, i.e. `-(~x)`.
+   - The expression `x**y**z` would be an implicit subexpression without spaces
+     containing more than one `**` operator. This code would be executed as `x**(y**z)`
+     following the implicit order, but the explicit parentheses should be included for
+     clarity.
+   - In the expression `t**v*x**y + z`, no spaces are used around the multiplication
+     operator due to the presence of the lower-priority addition operator. However,
+     this would lead to `t**v*x**y` being an implicit subexpression without spaces
+     containing more than one `**` operator. This code would be executed as 
+    `(t**v)*(x**y) + z`, but the explicit parentheses should be included for clarity.  
+6. When breaking lines at operators, always include the operator at the beginning
+   of the new line rather than at the end of the old line, and use parentheses if
+   needed to allow for implicit rather than explicit line breaks. For example, if
+   `x + y` must be broken, prefer
+   ```python3
+   (x
+    + y)
+   ```
+   over
+   ```python3
+   x + \
+   y
+   ```
```

Comment:
This is specified by PEP8; we might be able to remove it, or at least we should mention that it is just a reinforcement of PEP8.